Map/version.txt

```diff
@@ -1,2 +1,2 @@
-458
+459
```

Comment:
I think this would need to be 460 because there are two new items in the changelog